tests/acceptance/EnableAndDisableSignupConfirmationCest.php

```diff
@@ -42,12 +42,14 @@ private function setSignupConfirmationSetting(AcceptanceTester $I, $enabled) {
 
   private function countConfirmationEmails(AcceptanceTester $I) {
     $I->amOnUrl(AcceptanceTester::MAIL_URL);
+    $I->waitForElement('.messages.ng-scope'); // ensure that angular is loaded
```

Comment:
Spent some time looking for Angular in Mailpoet plugin 😄 Please update the note so that it is obvious that this happens in mailhog.
Or maybe we can create a helper `$I->amOnMailboxAppPage()` or something like that and add this waiting inside. I think we have this $I->amOnUrl(AcceptanceTester::MAIL_URL) on more places (since we created a constant for that)